esmvalcore/_config/extra_facets/ipslcm-mappings.yml

```diff
@@ -0,0 +1,239 @@
+# configuration is to avoid using it. You may use customized settings
+# by installing a modified version of this file as
+# ~/.esmvatlool/variable_details/native6-ipsl-cm6-*.yml
+#
+# Key 'positive' allows to tell ESMValTool when the sign convention
+# for a variable is reversed between IPSL-CM6 and CMIP6. In that case,
+# just e.g. put 'positive: down' if the CMIP6 convention is 'positive: up'
+#
+# The main key below, 'IPSL-CM6', is the one to use as a value for
+# attribute 'dataset' in the 'datasets' entry of recipes; it matches
+# the module name 'ipsl.py' in 'cmor_fixes/ipsl*/ipsl.py'
+#
+---
+#  A series of shortcuts for repetitive settings
+ShortCuts:
+  General: &gene {model: IPSLCM6, use_cdo: false}
+  ATM3DVARS: &atm3dvars {group: histmthNMC, dir: ATM, <<: *gene}
+  Atmvars: &atmvars {group: histmth, dir: ATM, <<: *gene}
+  SrfVars: &srfvars {group: sechiba_history, dir: SRF, <<: *gene}
+  StoVars: &stovars {group: stomate_history, dir: SBG, <<: *gene}
+  StiVars: &stivars {group: stomate_ipcc_history, dir: SBG, <<: *gene}
+  SechVars: &sechvars {group: sechiba_history, dir: SBG, <<: *gene}
+  OceTVars: &ocetvars {group: grid_T, dir: OCE, <<: *gene}
+  OceUVars: &oceuvars {group: grid_U, dir: OCE, <<: *gene}
+  OceVVars: &ocevvars {group: grid_V, dir: OCE, <<: *gene}
+  OceDvars: &ocedvars {group: diaptr, dir: OCE, <<: *gene}
+  OcePtr: &oceptr {group: ptrc_T, dir: BGC, <<: *gene}
+  IceVars: &icevars {group: icemod, dir: ICE, <<: *gene}
+
+
+IPSL-CM6:
+  # ========================================================================================
+  Amon:
+    # ========================================================================================
+    # ATM 3D Variables
+    ta: {ipsl_varname: ta, <<: *atm3dvars}
+    ua: {ipsl_varname: ua, <<: *atm3dvars}
+    va: {ipsl_varname: va, <<: *atm3dvars}
+    zg: {ipsl_varname: zg, <<: *atm3dvars}
+    hus: {ipsl_varname: hus, <<: *atm3dvars}
+    hur: {ipsl_varname: hur, <<: *atm3dvars}
+
+    # ATM general variables
+    pr: {ipsl_varname: precip, <<: *atmvars}
+    psl: {ipsl_varname: slp, <<: *atmvars}
+    tas: {ipsl_varname: t2m, <<: *atmvars}
+    ts: {ipsl_varname: tsol, <<: *atmvars}
+    huss: {ipsl_varname: q2m, <<: *atmvars}
+    uas: {ipsl_varname: u10m, <<: *atmvars}
+    "vas": {ipsl_varname: v10m, <<: *atmvars}
+    sfcWind: {ipsl_varname: wind10m, <<: *atmvars}
+    hurs: {ipsl_varname: rh2m, <<: *atmvars}
+
+    # ATM general variables
+    precip: {ipsl_varname: precip, <<: *atmvars}
+    prw: {ipsl_varname: prw, <<: *atmvars}
+    slp: {ipsl_varname: slp, <<: *atmvars}
+    t2m: {ipsl_varname: t2m, <<: *atmvars}
+    q2m: {ipsl_varname: q2m, <<: *atmvars}
+    u10m: {ipsl_varname: u10m, <<: *atmvars}
+    v10m: {ipsl_varname: v10m, <<: *atmvars}
+    wind10m: {ipsl_varname: wind10m, <<: *atmvars}
+
+    #  -> Turbulent fluxes
+    taux: {ipsl_varname: taux, <<: *atmvars}
+    tauy: {ipsl_varname: tauy, <<: *atmvars}
```

Comment:
What's the difference between these turbulent fluxes and the ones below (~ lines 104ff)?

Reply:
same issue as for rsds et al. Moved to a dedicated section